src/architecture/msgPayloadDefCpp/THROutputMsgPayload.h

```diff
@@ -30,7 +30,9 @@ THROutputMsgPayload
 //@endcond
 {
     double maxThrust;                    //!< N  Steady state thrust of thruster
-    double thrustFactor;                 //!< -- Current Thrust Percentage
+    double thrustFactor;                 //!< -- Current thrust percentage due to ramp up/down
```

Comment:
please remove the `--` lines at beginning of unites variable definitions

Reply:
Agreed, updating full file comments to be consistent with other header files.